terraform/gen.go

```diff
@@ -634,6 +638,16 @@ variable "google_service_account" {
   type        = string
 }
 
+variable "google_github_private_keys_bucket" {
+  description = "Name of the bucket for github private keys"
+  type        = string
+}
+
+variable "google_kms_github_private_key" {
```

Comment:
```suggestion
variable "google_kms_github_private_key_name" {
```
Otherwise it sounds as if something unsafe happens here (e.g. key is inside that env).

Reply:
Done.